app/views/api/integrations/apicast/shared/_mapping_rules.html.slim

```diff
@@ -9,26 +9,53 @@
           - if @service.using_proxy_pro?
             th Redirect
           th title="increment" +
-          th colspan="2"
+          th
             ' Metric or Method
             | (
             = link_to 'Define', admin_service_metrics_path(@service), title: 'Define Methods & Metrics for this service'
             | )
+          th colspan="2"
+            ' Last?
 
-      tbody
+      tbody#sortable.ui-sortable
         - metrics = options_from_collection_for_select(@service.metrics, 'id', 'name')
 
-        - @proxy.proxy_rules.each do |rule|
-          tr
-            = render 'api/integrations/apicast/shared/mapping_rule', form: f, rule: rule, metrics: metrics
-        tr#new-proxy-rule-template style="display:none;"
+        - @proxy.proxy_rules.ordered.each_with_index do |rule, i|
```

Comment:
Maybe an instance variable in the controller or better a presenter?

Reply:
I'm not convinced this deserves a presenter given the short life expectancy of this integration page. Any other way you might suggest to change this?

Reply:
OK